assets/dev/js/editor/components/ai/components/prompt-action-selection.js

```diff
@@ -0,0 +1,50 @@
+import { MenuItem, Select, FormControl, InputLabel } from '@elementor/ui';
+
+const labelToDashCash = ( str ) => str.toLowerCase().replace( /\s/g, '-' );
```

Comment:
```suggestion
const slugify = ( str ) => str.toLowerCase().replace( /\s/g, '-' );
```

maybe?

Reply:
I think that it is more clear: labelToDashCash..